api/pkg/apis/projectcalico/v3/felixconfig.go

```diff
@@ -835,6 +835,10 @@ type FelixConfigurationSpec struct {
 	// FlowLogGoldmaneServer is the flow server endpoint to which flow data should be published.
 	FlowLogsGoldmaneServer *string `json:"flowLogsGoldmaneServer,omitempty"`
 
+	// FlowLogsLocalSocket configures local unix socket for reporting flow data from each node. [Default: Disabled]
+	// +kubebuilder:validation:Enum=Disabled;Enabled
+	FlowLogsLocalSocket *string `json:"flowLogsLocalSocket,omitempty"`
```

Comment:
maybe we should name this `FlowLogsLocalReporter` or `FlowLogsNodeReporter`.

Reply:
Yeah, that makes sense to me - I like the `Local` terminology rather than `Node` myself.